### PR TITLE
Edits to cli_operations doc

### PR DESCRIPTION
**Pull request by bfallonf**
PR in regards to this dev PR: https://github.com/openshift/openshift-docs/pull/332

Fixed the "Common CLI Operations" table and formatting for the whole doc.

### CODE REVIEW

File: cli_reference/basic_cli_operations.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ This topic provides information on some general CLI operations and their syntax.
 You must link:get_started_cli.html[setup and login] with the CLI before you can
 perform these operations.
 
-== Common Operations
+== Common operations
```
Comment (adellape) [inline cli_reference/basic_cli_operations.adoc]: Are we no longer doing title case? And if not, I see you left title case in the table headings; intentional?

**Comment by bfallonf** [comment]
@adellape Ugh, sorry. I got confused for a second. Title case fixed now.

File: cli_reference/basic_cli_operations.adoc [hunk 2]
```diff
@@ -172,13 +172,16 @@ OpenShift objects at the project level.
...
+$ osadm new-project <project_name> --display-name=<display_name>
+--description=<description> --admin=<admin_username>
+--node-selector=<node_label_selector>
+----
 
 The following example creates a new project called `test` that appears in the
-Management Console as "Openshift 3 Sample", with `test-admin` as the project
-admin and launches any pods onto nodes matching label `environment : test`.
+Management Console as "OpenShift 3 Sample" with `test-admin` as the project
+administrator. Any pods are launched onto nodes with the matching label
```
Comment (tpoitras) [inline cli_reference/basic_cli_operations.adoc]: Passive voice.

Could you write this instead?

"Pods launch onto nodes with the matching label...."

What's the purpose of saying "any"?

**Comment by tpoitras** [comment]
Just one minor thing. Other than that, LGTM. :+1:

**Comment by bfallonf** [comment]
@tpoitras Cool. Thanks for that. Changed the whole para to:

"The following example creates a new project called test that appears in the Management Console as "OpenShift 3 Sample". test-admin is the project administrator and, when launched onto nodes, pods receive the matching label of environment : test:"

**Comment by tpoitras** [comment]
It's not wrong, and I've done this too, but is it weird or bad style for us to begin a sentence with a file name or sys item name that is lower case? It looks wrong to me, but I don't know if it's worth always re-wording around such things so you never start a sentence with a lower-case file name.

**Comment by bfallonf** [comment]
@tpoitras At first I agreed with you, and made some changes, but then i saw the end result and liked it less so changed it back. I think it's ok how it is. If there wasn't markup around the test-admin then it might be a different story, but as there is, it looks like 'test-admin' and I think that makes it ok. 

If anyone disagrees let me know.

**Comment by adellape** [comment]
LGTM, merging.

**Pull request merged**